packages/good-design/src/theme/hoc/withTheme.tsx

```diff
@@ -9,7 +9,8 @@ interface IWithThemeOpts {
 
 export const withTheme =
   (options?: IWithThemeOpts) =>
-  <T,>(Component: React.ComponentType<T>) => {
+  // @ts-ignore
+  <T,>(Component: React.ComponentType<T & { children?: React.ReactNode }> & { children?: React.ReactNode }) => {
```

Comment:
there should be generics like PropsWIthChildren etc. please look up how to describe it correctly using the types react provides

Reply:
Ah that looks a lot cleaner yeah, updated now